README.md

```diff
@@ -42,323 +59,28 @@ Migrating data from Conjur Open Source to Conjur EE is simple using our
 
 [migration]: design/MIGRATION.md
 
-# Development
```

Comment:
We should probably leave this section in and link it to contributing header. Not everyone that wants to work on the code wants to do it to contribute the code back.

Reply:
the CONTRIBUTING file is still linked to, it's just lower and in a section called `Contributing` consistent with the rest of our repos

take a look there and see if it could be clearer about telling people where to look for development instructions, I'm happy to edit it if you think it could be better.

Reply:
I think from my perspective, as long as there's some indication that there's "development" info in the contributing doc it's fine with me.

Reply:
updated the comment so that it indicates it includes "general development instructions"